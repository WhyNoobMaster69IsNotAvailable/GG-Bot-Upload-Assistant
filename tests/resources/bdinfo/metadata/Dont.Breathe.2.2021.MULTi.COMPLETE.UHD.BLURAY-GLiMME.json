{
    "largest_playlist": "00001.MPLS",
    "raw_video_file": "/data/downloads/FullDisk/Dont.Breathe.2.2021.MULTi.COMPLETE.UHD.BLURAY-GLiMMER/BDMV/STREAM/00001.m2ts",
    "raw_file_name": "Dont.Breathe.2.2021.MULTi.COMPLETE.UHD.BLURAY-GLiMMER",
    "mediainfo": "/app/temp_upload/1e7294d647c956dc621f05abe85a0081b792a5ab25728ea0cdc4313cc3f4fcd0/mediainfo.txt",
    "upload_media": "/data/downloads/FullDisk/Dont.Breathe.2.2021.MULTi.COMPLETE.UHD.BLURAY-GLiMMER/",
    "bdinfo_output_split": "[\"b'Please\", 'wait', 'while', 'we', 'scan', 'the', 'disc...\\\\n#', 'Group', 'Playlist', 'File', 'Length', 'Estimated', 'Bytes', 'Measured', 'Bytes', '\\\\n\\\\n1', '1', '00001.MPLS', '01:38:14', '54,294,134,784', '-', '\\\\n2', '2', '00373.MPLS', '00:05:07', '739,196,928', '-', '\\\\n3', '2', '00380.MPLS', '00:05:07', '739,196,928', '-', '\\\\n4', '3', '00371.MPLS', '00:04:46', '687,409,152', '-', '\\\\n5', '3', '00378.MPLS', '00:04:46', '687,409,152', '-', '\\\\n6', '4', '00102.MPLS', '00:03:36', '7,698,432', '-', '\\\\n7', '5', '00372.MPLS', '00:03:14', '467,927,040', '-', '\\\\n8', '5', '00379.MPLS', '00:03:14', '467,927,040', '-', '\\\\n9', '6', '00374.MPLS', '00:00:56', '135,954,432', '-', '\\\\n10', '6', '00381.MPLS', '00:00:56', '135,954,432', '-', \"\\\\n'\"]"
}