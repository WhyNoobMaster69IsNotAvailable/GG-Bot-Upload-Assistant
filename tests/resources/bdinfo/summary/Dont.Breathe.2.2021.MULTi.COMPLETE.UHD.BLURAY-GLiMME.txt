
QUICK SUMMARY:

Disc Title: Don't Breathe 2 - 4K Ultra HD
Disc Label: Dont.Breathe.2.2021.MULTi.COMPLETE.UHD.BLURAY-GLiMMER
Disc Size: 56,686,185,102 bytes
Protection: AACS2
Playlist: 00001.MPLS
Size: 54,294,134,784 bytes
Length: 1:38:14.888
Total Bitrate: 73.68 Mbps
Video: MPEG-H HEVC Video / 57867 kbps / 2160p / 23.976 fps / 16:9 / Main 10 @ Level 5.1 @ High / 10 bits / HDR10 / BT.2020
Audio: English / Dolby TrueHD/Atmos Audio / 7.1 / 48 kHz /  4174 kbps / 24-bit (AC3 Embedded: 5.1 / 48 kHz /   640 kbps / DN -31dB)
Audio: English / Dolby Digital Audio / 5.1 / 48 kHz /   640 kbps / DN -31dB
Audio: Hindi / Dolby Digital Audio / 5.1 / 48 kHz /   640 kbps / DN -31dB
Audio: Polish / Dolby Digital Audio / 5.1 / 48 kHz /   640 kbps / DN -31dB
Audio: Spanish / DTS-HD Master Audio / 5.1 / 48 kHz /  2234 kbps / 16-bit (DTS Core: 5.1 / 48 kHz /  1509 kbps / 16-bit)
Audio: Tamil / Dolby Digital Audio / 5.1 / 48 kHz /   640 kbps / DN -31dB
Audio: Telugu / Dolby Digital Audio / 5.1 / 48 kHz /   640 kbps / DN -31dB
Audio: English / Dolby Digital Audio / 2.0 / 48 kHz /   192 kbps / DN -31dB
Audio: English / Dolby Digital Audio / 2.0 / 48 kHz /   192 kbps / DN -31dB
Subtitle: English / 8.750 kbps
Subtitle: English / 13.764 kbps
Subtitle: Arabic / 5.802 kbps
Subtitle: Bulgarian / 8.517 kbps
Subtitle: Croatian / 8.216 kbps
Subtitle: Czech / 8.174 kbps
Subtitle: Greek / 9.221 kbps
Subtitle: Hebrew / 6.839 kbps
Subtitle: Hungarian / 9.157 kbps
Subtitle: Icelandic / 8.762 kbps
Subtitle: Polish / 7.960 kbps
Subtitle: Portuguese / 8.951 kbps
Subtitle: Romanian / 8.706 kbps
Subtitle: Serbian / 8.583 kbps
Subtitle: Slovak / 7.759 kbps
Subtitle: Slovenian / 7.576 kbps
Subtitle: Spanish / 8.940 kbps
Subtitle: Turkish / 9.013 kbps
Subtitle: English / 32.401 kbps
Subtitle: Spanish / 29.056 kbps
Subtitle: English / 52.669 kbps
Subtitle: Spanish / 57.085 kbps

<---- END FORUMS PASTE ---->