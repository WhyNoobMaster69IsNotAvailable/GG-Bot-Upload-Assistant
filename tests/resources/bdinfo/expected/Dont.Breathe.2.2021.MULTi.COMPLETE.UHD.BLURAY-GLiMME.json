{
    "video": [
        {
            "codec": "HEVC",
            "bitrate": "57867 kbps",
            "resolution": "2160p",
            "fps": "23.976 fps",
            "aspect_ratio": "16:9",
            "profile": "Main 10 @ Level 5.1 @ High",
            "bit_depth": "10 bits",
            "dv_hdr": "HDR10",
            "color": "BT.2020"
        }
    ],
    "audio": [
        {
            "language": "English",
            "atmos": "Atmos Audio",
            "codec": "Dolby TrueHD",
            "channels": "7.1",
            "sample_rate": "48 kHz",
            "bitrate": "4174 kbps",
            "bit_depth": "24-bit"
        },
        {
            "language": "English",
            "codec": "Dolby Digital Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "640 kbps",
            "bit_depth": "DN -31dB"
        },
        {
            "language": "Hindi",
            "codec": "Dolby Digital Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "640 kbps",
            "bit_depth": "DN -31dB"
        },
        {
            "language": "Polish",
            "codec": "Dolby Digital Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "640 kbps",
            "bit_depth": "DN -31dB"
        },
        {
            "language": "Spanish",
            "codec": "DTS-HD Master Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "2234 kbps",
            "bit_depth": "16-bit"
        },
        {
            "language": "Tamil",
            "codec": "Dolby Digital Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "640 kbps",
            "bit_depth": "DN -31dB"
        },
        {
            "language": "Telugu",
            "codec": "Dolby Digital Audio",
            "channels": "5.1",
            "sample_rate": "48 kHz",
            "bitrate": "640 kbps",
            "bit_depth": "DN -31dB"
        },
        {
            "language": "English",
            "codec": "Dolby Digital Audio",
            "channels": "2.0",
            "sample_rate": "48 kHz",
            "bitrate": "192 kbps",
            "bit_depth": "DN -31dB"
        },
        {
            "language": "English",
            "codec": "Dolby Digital Audio",
            "channels": "2.0",
            "sample_rate": "48 kHz",
            "bitrate": "192 kbps",
            "bit_depth": "DN -31dB"
        }
    ],
    "title": "Don't Breathe 2 - 4K Ultra HD",
    "label": "Dont.Breathe.2.2021.MULTi.COMPLETE.UHD.BLURAY-GLiMMER",
    "size": 52.79312385432422,
    "playlist": "00001.MPLS",
    "length": "1:38:14"
}
